thermo_nerf/rgb_concat/concat_nerfacto_model.py

```diff
@@ -221,7 +222,7 @@ def get_metrics_dict(self, outputs, batch):
         metrics_dict = {}
         gt_rgb = batch["image"].to(self.device)  # RGB or RGBA image
 
-        predicted_rgb = outputs["rgb"]
+        predicted_rgb = outputs[RenderedImageModality.RGB.value]
```

Comment:
To have the predicted rgb images, we could only use `output["rgb"]` and not `output["img"]`. Besides, we could not change it to `output["img"]` as it is created in nerfstudio.